session/session-layer.go

```diff
@@ -106,6 +107,8 @@ func (s *Layer) receiveThread() {
 				return
 			}
 
+			fmt.Printf("Received: 0x%X\n", frameIn.Payload)
```

Comment:
Do we want some kind of leveled logging? logrus for example. This seems pretty low-level debug outout...

Reply:
See #9